### PR TITLE
Update package.json dependencies

### PR DESCRIPTION
**Pull request by ryanseys**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,10 @@
       "email": "pcostell@google.com"
     },
     {
+      "name": "Ryan Seys",
+      "email": "ryan@ryanseys.com"
+    },
```
Comment (stephenplusplus) [inline package.json]: Who's this guy?!

**Comment by stephenplusplus** [comment]
The only thing that will actually be changed was `extend` having a major bump from 1 to 2.0. The change was minimal though, according to their changelog. Merge city, here we come.

File: package.json (hunk 1 above)
Comment (ryanseys) [inline package.json]: Am I worthy yet??

**Comment by ryanseys** [comment]
Ah, true dat. Didn't think of this. Just used `npm-check-updates` to bump up the jam all over.

File: package.json (hunk 1 above)
Comment (stephenplusplus) [inline package.json]: Of course! :dancer: :lollipop:

**Pull request merged**